KeePassFaviconDownloaderExt.cs

```diff
@@ -439,13 +440,22 @@ private Uri getFromFaviconExplicitLocation(Uri fullURI, ref MemoryStream ms, ref
         /// <param name="message">Any error message is sent back through this string.</param>
         /// <returns></returns>
         private bool getFavicon(Uri uri, ref MemoryStream ms, ref string message)
+        {
+            return getFaviconWithSecurityProtocol(uri, ref ms, ref message, originalSecurityProtocol) ||

```

Comment:
If originalSecurityProtocol is Tls11 or Tls12 but the attempt to get the favicon fails for a reason unrelated to TLS version, it looks like we will now make subsequent unnecessary attempts. We should instead ensure that we don't try the same protocol more than once.